### PR TITLE
Metadata Export and Citations

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

File: js/packages/glimmer-idc/src/utils/results.ts [hunk 1]
```diff
@@ -106,6 +110,19 @@ export class ResultsService {
       this.sortOrder = `&sort_order=${queryParams.get('sort_order')}`;
     }
 
+    // TODO: Can we simply inspect the URL path for `/node/<node_id>`?
+    if (queryParams.has('nodeId')) {
+      console.log('  >> Found nodeId: ' + queryParams.get('nodeId'));
```
Comment (jabrah) [inline js/packages/glimmer-idc/src/utils/results.ts:115]: Should remove console.log (I think that was my bad)

File: js/packages/glimmer-idc/src/components/title-bar.ts [hunk 2]
```diff
@@ -15,9 +15,40 @@ interface Args {
...
   hasAdvancedSearch: boolean;
+  query: string;
+  searchType: string;
 }
 
 export default class TitleBar extends Component<Args> {
+  @tracked isAuthenticated: boolean = false;
+
+  get itemsExportUrl() {
+    return `export_items?${this.args.query}`;
```
Comment (jabrah) [inline js/packages/glimmer-idc/src/components/title-bar.ts:26]: URL should have a `/` at the start

File: js/packages/glimmer-idc/src/components/title-bar.ts [hunk 3]
```diff
@@ -15,9 +15,40 @@ interface Args {
...
+  }
+
+  get collectionsExportUrl() {
+    return `export_collections?${this.args.query}`;
```
Comment (jabrah) [inline js/packages/glimmer-idc/src/components/title-bar.ts:30]: Same as above, should have a `/` at the start

**Review by jabrah — CHANGES_REQUESTED** [review]

File: js/packages/glimmer-idc/src/components/title-bar.ts [hunk 4]
```diff
@@ -15,9 +15,40 @@ interface Args {
...
+    super(owner, args);
+
+    this.fetchUser();
+  }
+
+  async fetchUser() {
+    let res: Response = await fetch('/jsonapi');
+    let data: JsonApiUserResponse = await res.json();
+
+    if (data && data.meta && data.meta.links && data.meta.links.me && data.meta.links.me.meta && data.meta.links.me.meta.id) {
```
[the remark this answers is not in the document]
Comment (jaredgalanis) [reply js/packages/glimmer-idc/src/components/title-bar.ts:47]: Will address this in follow up. I took a look at babel and most things needed seem to be in place, so I'm not sure why optional chaining would not work, but I did try it out.

**Review by jabrah — APPROVED** [review]

**Pull request merged**